### PR TITLE
UC6.4 new rev

### PR DESCRIPTION
**Pull request by eric-schuh**
Cleanup and updates from review w/Joe. Questions coming out of that review are:

1. The prose states the desire to present the credentials to the verifier automatically instead of delivering them “by hand”. However, the diagram shows the flow passing through the Applicant to confirm sending the two credentials to the verifier. This seems to be exactly the “by hand” that the prose is trying to avoid.

    1. So, what is being automated?
    1. Contacting service provider B after a request for one of their credentials?
    1. Providing the required information to service provider B to issue its credential?
    1. Providing the credentials A,B to the verifier once B is received by the holder?

    1. Contacting service provider B after a request for one of their credentials?
        1. How does the holder know where to go to get B?
            1. Does the wallet magically know already?
            1. Does the verifier specify it?
            1. How would this work when B can be secured from any number of issuers? Or even from the same issuer at different issuer apps? (The DID of the issuer may not be enough to reveal the service endpoint the user uses to get the credential).
How does the “Holder” know what to do when it gets to the Issuer of B?
            1. What’s the route/endpoint?
                1. What’s the UX expectation?
                    1. HTML page shown to user
                    1. Background service endpoint
                1. What’s the structure of the request that goes to the endpoint? 
                    1. How are parameters passed?
                    1. What parameters/data are required?
                    1. How does the user authenticate into the service?
                1. How does the user authorize the retrieval of B? 
                    1. If this part is automated, does that mean that the verifier triggers a complex interaction without the user’s involvement?
                    1. Is the authorization implicit in the forwarding of the A+B VPR? What if the user thinks they already have a B and did not intend to automate a provisioning ceremony to get a new B?
    1. Providing the required information to service provider B to issue its credential?
        1. It’s an open-ended question as to what provider B requires to issue a credential. Do you need to authenticate into an account? Do you need to upload a driver’s license? Do you need to sign a terms of service?
        1. How are the requirements to get B defined?
        1. How are they satisfied in an automated manner?
    1. Providing the credentials A,B to the verifier once B is received by the holder?
        1. The flow diagram suggests this is not automated, but the prose suggests it is.
        1. If it *is* automated, when is the consent to share B secured?

Image: 

image

***
<a href="https://pr-preview.s3.amazonaws.com/eric-schuh/vc-api-use-cases/pull/19.html" title="Last updated on Aug 30, 2022, 7:38 PM UTC (a70169e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api-use-cases/19/a163df1...eric-schuh:a70169e.html" title="Last updated on Aug 30, 2022, 7:38 PM UTC (a70169e)">Diff</a>

### CODE REVIEW

**Comment by eric-schuh** [comment]
@mavarley could you take a look at the questions? Would like to see your feedback as you submitted this use case.

**Comment by eric-schuh** [comment]
@mavarley ping in case you missed this

**Comment by eric-schuh** [comment]
Step 5 - change comma to semi-colon and add clearer language around "B was not found"

File: index.html [hunk 1]
```diff
@@ -411,33 +411,33 @@ <h3>Aggregated Credential Workflow</h3>
...
+              backgroundcheck.com (Issuer App)->>backgroundcheck.com (Issuer App):evaluate requested info
+              backgroundcheck.com (Issuer App)->>Generic Issuer (Issuer Service):genericissuer.com/credentials/issue?"backgrounCheckCredential+info"
+              Generic Issuer (Issuer Service)->>backgroundcheck.com (Issuer App):issued VC
+              backgroundcheck.com (Issuer App)->>Kenzie's Wallet (Holder Service):kenzieswallet.com/credentials/recieve?"VC"
+              Kenzie's Wallet (Holder Service)->>Kenzie (Holder):Notify B Ready
+              Kenzie (Holder)->>jobapplications.com (Verifier App):jobapplications.com/presenations/submit?"application"
+              jobapplications.com (Verifier App)->>Kenzie (Holder):Request Credentials A and B
+              Kenzie (Holder)->>Kenzie's Wallet (Holder Service):Send A and B to jobapplications.com
+              Kenzie's Wallet (Holder Service)->>jobapplications.com (Verifier App):jobapplications.com/presentations/submit?"application with A and B"
+              jobapplications.com (Verifier App)->>Generic Verifier (Verifier Service):genericVerifer.com/presentations/verify?"applicationwith A and B"
```
Comment (TallTed) [inline index.html]:
```suggestion
              jobapplications.com (Verifier App)->>Generic Verifier (Verifier Service):genericVerifer.com/presentations/verify?"application with A and B"
```

File: index.html [hunk 2]
```diff
@@ -411,33 +411,33 @@ <h3>Aggregated Credential Workflow</h3>
...
-              jobapplications.com - Verifier App->>jobapplications.com - Verifier App:process application     
-              jobapplications.com - Verifier App->>Kenzie's Wallet - Holder Service:Application Accepted     
-              Kenzie's Wallet - Holder Service->>Kenzie - Holder:Application Accepted 
+              sequenceDiagram
+                  autonumber
+              Kenzie (Holder)->>jobapplications.com (Verifier App):jobapplications.com/presenations/submit?"application"
+              jobapplications.com (Verifier App)->>jobapplications.com (Verifier App):process application
+              jobapplications.com (Verifier App)->>Kenzie (Holder):Request credentials A and B
+              Kenzie (Holder)->>Kenzie's Wallet (Holder Service):Check for A and B
+              Kenzie's Wallet (Holder Service)->>Kenzie (Holder):Display A, B not found
```
Comment (TallTed) [inline index.html]:
```suggestion
              Kenzie's Wallet (Holder Service)->>Kenzie (Holder):Display A; B not found
```

File: index.html (hunk 1 above)
Comment (eric-schuh) [reply index.html]: This item will be gotten with the full <br> update to the diagrams

File: index.html [hunk 3]
```diff
@@ -411,33 +411,33 @@ <h3>Aggregated Credential Workflow</h3>
...
+                      jobapplications.com <br/>(Verifier App)->>Kenzie <br/>(Holder):Request Credentials A and B
+                      Kenzie <br/>(Holder)->>Kenzie's Wallet <br/>(Holder Service):Send A and B to jobapplications.com
+                      Kenzie's Wallet <br/>(Holder Service)->>jobapplications.com <br/>(Verifier App):jobapplications.com/presentations/submit?"application with A and B"
+                      jobapplications.com <br/>(Verifier App)->>Generic Verifier <br/>(Verifier Service):genericVerifer.com/presentations/verify?"applicationwith A and B"
+                      Generic Verifier <br/>(Verifier Service)->>jobapplications.com <br/>(Verifier App):Verification Result
+                      jobapplications.com <br/>(Verifier App)->>jobapplications.com <br/>(Verifier App):process application
+                      jobapplications.com <br/>(Verifier App)->>Kenzie's Wallet <br/>(Holder Service):Application Accepted
+                      Kenzie's Wallet <br/>(Holder Service)->>Kenzie <br/>(Holder):Application Accepted
             </pre>
             <figcaption>UC2.4 Mermaid</figcaption>
```
Comment (TallTed) [inline index.html:442]:
```suggestion
            <figcaption>UC6.4 Mermaid</figcaption>
```

**Comment by TallTed** [comment]
@jandrieu -- Merged without correction to `figcaption`!

**Pull request merged**